internals/overlord/logstate/gatherer.go

```diff
@@ -383,9 +383,13 @@ func newLogClient(target *plan.LogTarget) (logClient, error) {
 			ScopeName:  cmd.ProgramName,
 		}), nil
 	case plan.SyslogTarget:
+		hostname, err := os.Hostname()
+		if err != nil {
+			return nil, fmt.Errorf("cannot get hostname: %w", err)
```

Comment:
I suppose it's probably better to not send the hostname and keep going in case of an error here (maybe in some container environments there's no hostname?) So let's log and continue:
```suggestion
			logger.Noticef("Cannot get hostname for syslog: %v", err)
			hostname = ""
```